app/assets/javascripts/selectizer.js.coffee

```diff
@@ -3,6 +3,10 @@ class Selectizer
     @$el = $(el)
     @dataAttr = @$el.attr('data-attr') || 'default_field'
 
+    # `required` inputs don't work with Selectize
```

Comment:
I think linking to the gh issue is great for documenation; summary of gh issue not necesssary